Mathport/Syntax/Transform/Tactic.lean

```diff
@@ -36,14 +36,52 @@ mathport_rules
 
 -- common obsolete patterns from haveI
 mathport_rules
-  | `(by have $hd:haveDecl; exact $t) => `(have $hd:haveDecl; $t)
-  | `(by have $hd:haveDecl <;> exact $t) => `(have $hd:haveDecl; $t)
+  | `(by have $hd:haveDecl
```

Comment:
Oops, did I forget a `$[;]?` here?  It's mildly annoying that sometimes atoms matter (like here) and sometimes they don't (like when `·` matches both `·` and `.`).